README.md

```diff
@@ -99,127 +99,142 @@ val config: Try[YourConfClass] = loadConfig[YourConfClass]
 
 Currently supported types for fields are:
 - `String`, `Boolean`, `Double` (standard
-and percentage format ending with `%`), `Float` (also supporting percentage),
-`Int`, `Long`, `Short`, `URL`, `Duration`, `FiniteDuration`
-- all collections implementing the `TraversableOnce` trait where the elements type
-is in this list
-- `Option` for optional values, i.e. value that can or cannot be in the configuration
-- `Map` with `String` keys and any value type that is in this list
-- everything in [`java.time`](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html).
- Because different formats are supported, converters for types that support multiple formats must be configured
- before they can be used (see [Configurable converters](#configurable-converters))
-- typesafe `ConfigValue`, `ConfigObject` and `ConfigList`
-- case classes
-- sealed families of case classes (ADTs)
+  and percentage format ending with `%`), `Float` (also supporting percentage),
+  `Int`, `Long`, `Short`, `URL`, `Duration`, `FiniteDuration`;
+- all collections implementing the `TraversableOnce` trait where the type of
+  the elements is in this list;
+- `Option` for optional values, i.e. values that can or cannot be in the configuration;
+- `Map` with `String` keys and any value type that is in this list;
+- everything in [`java.time`](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html) (must be
+  configured first - see [Configurable converters](#configurable-converters));
```

Comment:
I think people would follow the link after that, where the "configuring" part is explained in detail. As this is just a list of the supported types, I would like not to have extensive usage instructions here and instead point to the relevant sections in the README.